llvm/test/Transforms/ConstraintElimination/abs.ll

```diff
@@ -0,0 +1,39 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -passes=constraint-elimination -S %s | FileCheck %s
+
+define i1 @abs_int_min_is_not_poison(i32 %arg) {
```

Comment:
Could you submit the tests separately and also add a version with constant arguments (both positive and negative)?

Reply:
ping @fhahn - happy to send a separate PR with the tests, but need a clarification regarding "constant arguments" - what did you mean

Reply:
Constants to the llvm.abs calls